src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+public class Task {
+
+    /**Description of task*/
+    private String name;
+    /**records the completion of the task*/
+    private boolean isDone;
+    /**total number of tasks in the list*/
+    private static int noOfTasks = 0;
+    /**number of incomplete tasks*/
+    public static int noOfIncompleteTasks = 0;
```

Comment:
Perhaps NoOfIncompleteTasks would be a better name?

Reply:
Thank you for the suggestion! Should this variable name be in camel case?